department-of-veterans-affairs/caseflow
# adding unique key for child instance

Resolves [APPEALS-29283](https://jira.devops.va.gov/browse/APPEALS-29283)

# Description
Added key on different Page Route to remove the warning. 

## Acceptance Criteria
- [ ] Warning: Each child in a list should have a unique "key" prop. should not be present. 
- [ ] All the links should work as expected. 

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
2. login as ACBAUERVVHAH, redirect to localhost:3000. 
3. Press F12 and check in console and see if you are still seeing - '"Warning: Each child in a list should have a unique 'key' prop."
4.  check all links are working as expected. 

# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
